### PR TITLE
Adding two new Abilities for Succubus characters and the posibility of sitting on someones face

### PR DESCRIPTION
**Pull request by Vulpias**
Cameron helped me greatly with the coding of the Succubus abilities.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm [hunk 1]
```diff
@@ -796,3 +796,138 @@
...
+
+/datum/reagent/succubi_paralize/affect_blood(var/mob/living/carbon/M, var/alien, var/removed)
+	if(prob(7))
+		M.show_message("<span class='warning'>You lose sensation of your body.</span>")
+		M.Weaken(10)
+		return
+
+/mob/living/carbon/human/proc/face_sit()
+    set name = "Face Sit"
+    set desc = "Sit on your Preys Face"
```
Comment (ReddeyfishVor) [inline code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm:905]: small typo, should be "prey's"

File: code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm [hunk 2]
```diff
@@ -796,3 +796,138 @@
...
+
+    if(!istype(T) || T.isSynthetic())
+        to_chat(src, "<span class='warning'>\The [T] is not able to be sit on.</span>")
+        return
+
+    if(G.state != GRAB_AGGRESSIVE)
+        to_chat(C, "<span class='warning'>You must have the creature pinned on the ground to sit on them </span>")
+        return
+
+    src.visible_message("<font color='red'><b>[src] moves their ass to [T]'s head, sitting down on them, making them unable to see anything else than [src]'s butt </b></font>")
```
Comment (ReddeyfishVor) [inline code/modules/mob/living/carbon/human/species/station/station_special_abilities_vr.dm:932]: It seems as if sitting doesn't knock them down on the ground? So the target would still be standing up.

**Comment by ReddeyfishVor** [comment]
I don't know if we want facesitting to only be a trait. I think it makes more sense as something everyone can do inherently. See also https://github.com/CHOMPstation/CHOMPstation/blob/dc81a2856b576d8799c99a4f03390f4c0a8c1056/code/modules/mob/mob_grab.dm#L161 , which would be a better place to handle it.

**Comment by Vulpias** [comment]
a small bug, you need to have the Perk "Succubi drain" to be able to use the "Succubi Poison" perk

**Pull request merged**